src/Treeview.ts

```diff
@@ -38,7 +38,13 @@ const resourceIcons: [string, string][] = [
     ["Microsoft.Compute/virtualMachines/extensions", "extensions.svg"],
     ["Microsoft.Network/networkSecurityGroups", "nsg.svg"],
     ["Microsoft.Network/networkInterfaces", "nic.svg"],
-    ["Microsoft.Network/publicIPAddresses", "publicip.svg"]
+    ["Microsoft.Network/publicIPAddresses", "publicip.svg"],
+    ["Microsoft.DocumentDB/databaseAccounts", "documentdb.svg"],
+    ["Microsoft.Web/serverfarms", "appserviceplan.svg"],
+    ["Microsoft.Sql/servers", "sqldatabase.svg"],
+    ["databases", "database.svg"],
```

Comment:
I don't see databses or firewallrules in your screenshot. Is this really the correct type? Every other type has "Microsoft/"

Reply:
The resources are listed like this in my arm template, so assume it’s right. It may be because they’re bested and inherit the namespace from the parent

Reply:
Okay I looked through the quick start templates repo and I'm seeing the same thing: https://github.com/Azure/azure-quickstart-templates

So sounds good to keep the type name as-is, but I'd still like to have screenshots of all icons you're adding